src/debugging/coreclr/dockerDebugConfigurationProvider.ts

```diff
@@ -303,6 +300,34 @@ export class DockerDebugConfigurationProvider implements DebugConfigurationProvi
         return dockerfile;
     }
 
+    private async inferSsl(debugConfiguration: DockerDebugConfiguration, projectFile: string): Promise<boolean> {
+        if (debugConfiguration.configureSslCertificate !== undefined) {
+            return debugConfiguration.configureSslCertificate;
+        }
+
+        const launchSettingsPath = path.join(path.dirname(projectFile), 'Properties', 'launchSettings.json');
+
+        if (await this.fsProvider.fileExists(launchSettingsPath)) {
+            let launchSettingsString = await this.fsProvider.readFile(launchSettingsPath);
+            if (launchSettingsString.charCodeAt(0) === 0xFEFF) {
+                launchSettingsString = launchSettingsString.slice(1);
+            }
+            const launchSettings = JSON.parse(launchSettingsString);
+
+            //tslint:disable:no-unsafe-any no-any
+            if (launchSettings && launchSettings.profiles) {
+                const projectProfile = Object.values<any>(launchSettings.profiles).find(p => p.commandName === 'Project');
+
+                if (projectProfile && projectProfile.applicationUrl && /https:\/\//i.test(projectProfile.applicationUrl)) {
+                    return true;
+                }
+            }
+            //tslint:enable:no-unsafe-any no-any
+        }
```

Comment:
I can't decide if I want to automatically infer from launchSettings.json at this time. If we decide not to right now, we don't need a feature flag to guard this functionality, since activating it would require them to put the `configureSslCertificate` flag on their launch config.

Reply:
Additionally/alternatively, should we add a command to do it now?